color/src/parse.rs

```diff
@@ -543,6 +545,42 @@ pub fn parse_color(s: &str) -> Result<DynamicColor, ParseError> {
     }
 }
 
+impl FromStr for DynamicColor {
+    type Err = ParseError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        parse_color(s)
+    }
+}
+
+impl<CS: ColorSpace> FromStr for AlphaColor<CS> {
+    type Err = ParseError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        parse_color(s).map(DynamicColor::to_alpha_color)
+    }
+}
+
+impl<CS: ColorSpace> FromStr for OpaqueColor<CS> {
+    type Err = ParseError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        parse_color(s)
+            .map(DynamicColor::to_alpha_color)
+            .map(AlphaColor::discard_alpha)
```

Comment:
This one I'm less sure of. If the alpha isn't fully opaque, I'd maybe... expect this to error?